src/gmx/__init__.py

```diff
@@ -103,6 +103,194 @@ def run(work=None):
     return status
 
 
+def commandline_operation(executable=None, shell=False, arguments=None, input=None, output=None):
```

Comment:
`arguments` keyword needs to come second for the syntax we've been talking about. Move `shell` to the end.

```suggestion
def commandline_operation(executable=None, arguments=None, input=None, output=None, shell=False):
```

Reply:
done